server/src/test/java/io/crate/integrationtests/FulltextIntegrationTest.java

```diff
@@ -153,4 +153,12 @@ public void testCopyValuesFromStringArrayToIndex() throws Exception {
         execute("SELECT id, keywords FROM t_string_array WHERE keywords_ft = 'foo bar'");
         assertThat(response).hasRowCount(0L);
     }
+
+    @Test
+    public void test_can_add_geo_shape_array_column() throws Exception {
+        execute("create table t(id int)");
+        execute("alter table t add column keywords ARRAY(STRING) INDEX USING FULLTEXT");
+        execute("insert into t(id, keywords) VALUES (1, ['foo bar'])");
+        assertThat(response.rowCount()).isEqualTo(1L);
```

Comment:
Currently fails with `Cannot cast expressions from type `text_array` to type `text``